pkg/util/scrubber/default.go

```diff
@@ -94,11 +94,17 @@ func AddDefaultReplacers(scrubber *Scrubber) {
 		Regex: regexp.MustCompile(`(?i)([a-z][a-z0-9+-.]+://|\b)([^:]+):([^\s|"]+)@`),
 		Repl:  []byte(`$1$2:********@`),
 	}
-	passwordReplacer := matchYAMLKeyPart(
+	yamlPasswordReplacer := matchYAMLKeyPart(
 		`(pass(word)?|pwd)`,
 		[]string{"pass", "pwd"},
 		[]byte(`$1 "********"`),
 	)
+	// replace all alphanum + special chars except quotes
+	passwordReplacer := Replacer{
+		// see test cases for match patterns
+		Regex: regexp.MustCompile(`(?i)(\"?(?:pass(?:word)?|pswd|pwd)\"?)( = |=|: \"?)([0-9A-Za-z#!$%&()*+,\-./:<=>?@[\\\]^_{|}~]+)`),
```

Comment:
Why not just use <everything but space characters> for the password ?
```suggestion
		Regex: regexp.MustCompile(`(?i)(\"?(?:pass(?:word)?|pswd|pwd)\"?)( = |=|: \"?)([[:^space:]]+)`),
```

Reply:
I'm not familiar with `:space` but should it be `[^:space:]`?  Also, do we need the extra brackets?

Reply:
https://pkg.go.dev/regexp/syntax

```
[[:^alpha:]]   negated ASCII character class
```

Reply:
Excluding quotes and semi-colon is on purpose